xayagame/Makefile.am

```diff
@@ -5,11 +5,12 @@ EXTRA_DIST = rpc-stubs/game.json rpc-stubs/xaya.json
 
 libxayagame_la_CXXFLAGS = \
   $(JSONCPP_CFLAGS) $(JSONRPCCLIENT_CFLAGS) $(JSONRPCSERVER_CFLAGS) \
-  $(GLOG_CFLAGS) $(SQLITE3_CFLAGS) $(ZMQ_CFLAGS)
+  $(GLOG_CFLAGS) $(SQLITE3_CFLAGS) $(ZMQ_CFLAGS) -Wno-deprecated
```

Comment:
Instead of adding `-Wno-deprecated` here, please add it in `configure.ac` with the same `AX_CHECK_COMPILE_FLAGS` lines that add, for instance, `-Wall`.